packages/app/src/dimension-mapper/DimensionMapper.tsx

```diff
@@ -48,10 +48,13 @@ function DimensionMapper(props: Props) {
             <SlicingSlider
               key={`${index}`} // eslint-disable-line react/no-array-index-key
               dimension={index}
-              slicingIndex={val}
               maxIndex={rawDims[index] - 1}
-              mapperState={mapperState}
-              onChange={onChange}
+              initialValue={val}
```

Comment:
`SlicingSlider` is now "uncontrolled". It manages its own internal state (initialised with `initialValue`), which it updates instantly whenever the user interacts with the slider. Debouncing is now performed on the `onChange` callback, which is only called 250ms after the user has finished interacting with the slider.